doc/09-object-types.md

```diff
@@ -1470,6 +1470,11 @@ Cleanup Items:
   servicechecks\_age              | Duration              | **Optional.** Max age for servicechecks table rows (start\_time). Defaults to 0 (never).
   systemcommands\_age             | Duration              | **Optional.** Max age for systemcommands table rows (start\_time). Defaults to 0 (never).
 
+> **Supported units**
+>
+> Supported suffixes include ms (milliseconds), s (seconds), m (minutes), h (hours) and d (days).
+> Check the [language reference](17-language-reference.md#numeric-literals-).
```

Comment:
Should probably link `duration-literals` instead of `numeric-literals`. Also, the trailing `-` seems to be something added by GitHub, it should probably use the version without that to work on https://icinga.com/docs/icinga-2/:

https://github.com/Icinga/icinga2/blob/4904005378bf49b8b18abc6b7f2cb4e6ccc55f1b/doc/17-language-reference.md#L64

* https://icinga.com/docs/icinga-2/latest/doc/17-language-reference/#duration-literals works
* https://icinga.com/docs/icinga-2/latest/doc/17-language-reference/#duration-literals- doesn't

Reply:
as desired, the change was made to duration-literals